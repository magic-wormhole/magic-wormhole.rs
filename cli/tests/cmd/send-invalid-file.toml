bin.name = "wormhole-rs"
args = "send INVALID_FILE"
status.code = 1
