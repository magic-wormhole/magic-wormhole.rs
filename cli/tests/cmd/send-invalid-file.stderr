...
Error: 
   0: [91mINVALID_FILE does not exist[0m
...